README.md

```diff
@@ -71,7 +73,8 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+It should be put in the model.  Our controller should take parameters and return HTML or other web formats.  The model encapsulates the domains
+data and logic so any code we put into it can be incorporated into any view.
 ```
 
```

Comment:
In this example, the code would make the most sense as a Helper method, because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model methods concern how data goes to and comes from the database.